### PR TITLE
Enable more e2e tests for light-mode

### PR DESCRIPTION
**Pull request by piotr-iohk**
- d78a2f1ba1796b845bcef9c671919e4763e6c91b
  Enable wallet management tests for light-mode
  
- 5b8594a23497a2aef925805375d105e47abebc3c
  Add basic wallet restoration tests for light-mode
  
- 742677116a82fda4098fb3573257596aea60214c
  Test light-mode wallet restoration with small wallet
  

### Comments

With that we'll have 70 test cases running against light-mode.

Test pass on all 3 platforms:
 - [Linux](https://github.com/input-output-hk/cardano-wallet/runs/7542519937?check_suite_focus=true) :heavy_check_mark: 
 - [MacOS](https://github.com/input-output-hk/cardano-wallet/runs/7542523779?check_suite_focus=true) :heavy_check_mark:
 - [Windows](https://github.com/input-output-hk/cardano-wallet/runs/7542529152?check_suite_focus=true) :heavy_check_mark:

### Issue Number

ADP-1508 / ADP-2013

### CODE REVIEW

File: test/e2e/spec/lightmode_spec.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+    it "I can restore empty wallet" do
+      wid = create_shelley_wallet
+      wait_for_shelley_wallet_to_sync(wid)
+      wallet = SHELLEY.wallets.get(wid)
+      expect(wallet['balance']['total']['quantity']).to eq 0
+    end
+
+    it "I can restore wallet with funds and I see ADA balance" do
+      wid = create_fixture_wallet(:shelley)
+      wait_for_shelley_wallet_to_sync(wid)
```
Comment (piotr-iohk) [inline test/e2e/spec/lightmode_spec.rb:16]: This fixture wallet has ~18k transactions and growing. It seems that syncing it in light mode takes actually more time than syncing in full-mode (not sure if it is going to be feasible test then, need to test more)

Comment (HeinrichApfelmus) [reply test/e2e/spec/lightmode_spec.rb:16]: I can't find it right now, but when designing light mode, I did find out that it's fair to consider a person that does 200 transactions per year a day trader. 🤓 Light-mode won't be useful for people who do more transactions than this.

Comment (piotr-iohk) [reply test/e2e/spec/lightmode_spec.rb:16]: Ok, I've added small fixture wallet (only 1tx) for recurring e2e test for now.

**Review by HeinrichApfelmus — APPROVED** [review]
Looks good to me! 😊

**Pull request merged**